gym/envs/box2d/car_racing.py

```diff
@@ -183,6 +193,22 @@ def _destroy(self):
         self.road = []
         self.car.destroy()
 
+    def _init_colors(self):
+        if self.hardcore:
+            # domain randomize the bg and grass colour
+            self.norm_road_color = self.np_random.uniform(0, 0.8, size=3)
+
+            self.bg_color = self.np_random.uniform(0, 210, size=3)
+
+            self.grass_color = np.copy(self.bg_color)
+            idx = self.np_random.integers(3)
+            self.grass_color[idx] += 20
+        else:
+            # default colours
+            self.norm_road_color = np.array([0.4, 0.4, 0.4])
```

Comment:
Do we have to store the colors with two different representations? (one is 0-1, the other is 0-255)

Reply:
Gotcha, fixed it